rocketpy/tools.py

```diff
@@ -522,6 +523,69 @@ def normalize_quaternions(quaternions):
     return q_w / q_norm, q_x / q_norm, q_y / q_norm, q_z / q_norm
 
 
+def export_sensors_measured_data(sensor, filename, file_format, data_labels):
```

Comment:
I don't understand why the need to move this into a `tools.py` function. It is currently only used inside the `Sensor` class, right? Seems like a good `method` for the `Sensor` base class.

The way it was implemented before seemed alright. My only comment is that the method name should be `export_sensor_measured_data` (in the singular).

Reply:
Previously he was duplicating the code for different classes of sensors. 

I think the best way to define this at the parent (Sensor) class.